src/udp.cpp

```diff
@@ -54,6 +54,7 @@ UDP::UDP(uint16_t dport, uint16_t sport)
 : header_() {
     this->dport(dport);
     this->sport(sport);
+    this->auto_set_checksum = true;
```

Comment:
Move this to the initialization list. That's the best way to do it and it is done in this constructor for `header_`. Upstream is a little inconsistent, as seen in the IP constructors.

Reply:
done